src/drive/web/modules/views/OnlyOffice/helpers.js

```diff
@@ -4,3 +4,22 @@ export const isOnlyOfficeEnabled = () => flag('drive.onlyoffice.enabled')
 
 export const makeOnlyOfficeFileRoute = (file, isWithRouter) =>
   isWithRouter ? `/onlyoffice/${file.id}` : `/#/onlyoffice/${file.id}`
+
+export const isOnlyOfficeReadOnly = data =>
```

Comment:
export const isOnlyOfficeReadOnly = ({data}) => data.attributes.onlyoffice.editor.mode === 'view'

like that we avoid data.data